apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -25,10 +25,15 @@ function SectionProgressSelector({
   setShowProgressTableV2,
   progressTableV2ClosedBeta,
   sectionId,
+  hasSeenProgressTableInvite,
 }) {
   const [hasJustSwitchedToV2, setHasJustSwitchedToV2] = React.useState(false);
+  const [hasJustSwitchedToV1, setHasJustSwitchedToV1] = React.useState(false);
```

Comment:
Are `hasJustSwitchedToV2` and `hasJustSwitchedToV1` basically the same? Could we just use one and name it `hasJustToggledViews` or something?

Reply:
Good point.  In my mind, I liked the clarity of knowing which thing was toggled, but at the same time, I think it was false clarity since we never unset the opposite state when toggling.  I made this change.